src/test_suite.rs

```diff
@@ -67,13 +69,19 @@ impl Test {
             }
             Box::new([wasmtime::Val::I32(0)])
         });
+        let elapsed = now.elapsed();
+        // Convert the elapsed time to milliseconds
+        // Seems hacky, might need refactoring
+        let elapsed_in_ms = elapsed.as_secs_f32() * 1000.0;
```

Comment:
elapsed.as_millis();

Reply:
`as_millis()` yields an int, and most of the tests complete for less than 1ms and the results is 0ms. Only `as_secs` has a f32/f64 option.

Reply:
Another way that looks less hacky
```
let millis = elapsed.as_millis();
let micros = elapsed.as_micros();
format!("{} - {:?}.{:?}ms", self.name.clone(), millis, micros);
```

Reply:
Maybe you can move `1000.0` at the top as a const, something like `SECS_TO_MILLIS`, but I don't think it's a big deal.
You can also join the above two rows.

Reply:
Maybe we could extract the logic into a trait for the std::time::Duration struct and add a method `as_millis_f64()`